wp-content/themes/beyond/template-table.php

```diff
@@ -0,0 +1,83 @@
+          <rect x="0.5" y="0.809814" width="29" height="29" rx="14.5" stroke="#10355F" stroke-opacity="0.9"/>
+        </svg>
+      </a>
+    </td>
+    <td style="width: 55%;"><?php the_title(sprintf('<h2 class="entry-title featured-title"><a href="%s" rel="bookmark">', esc_url(get_permalink())), '</a></h2>'); ?>
+      <div class="related-analysis">
+        <?php foreach ($related_analysis_table as $id) { ?>
+          <?php $analysis = get_post($id);   
+          if ( $analysis ) { ?>
+            <h3 class="related-heading">Related Analysis:</h3>
+            <header class="entry-header living-header">
+            <h3 class="entry-title featured-title"><a href="<?php echo esc_url(get_permalink($analysis)); ?>" rel="bookmark"><?php echo get_the_title($analysis); ?></a></h3>
+            </header>
+            <div class="entry-summary">
+              <span class="excerpt-date"><?php echo get_the_date('l, F j, Y', $analysis); ?> </span>
+            </div>
+          <?php } //close if?>
+        <?php } //close foreach ?>
+        <?php //wp_reset_postdata();?>
+      </div> <!-- .related-analysis -->
+    </td>  
+  <?php } else { ?>
+    <td></td>
+    <td style="width: 55%;"><?php the_title(sprintf('<h2 class="entry-title featured-title"><a href="%s" rel="bookmark">', esc_url(get_permalink())), '</a></h2>'); ?></td>
+  <?php } // end if?>
+    <td style="width: 20%;"><?php echo get_field('location-region', $post->ID) ?></td>
+    <td style="width: 20%;"><?php the_modified_date(); ?></td>
+<?php } ?>
+
+<?php 
+  if ( isset( $_GET['scale'] ) ) {
+     // Grab the value selected in the dropdown from the URL
+    $filter_by_scale = rawurldecode( $_GET['scale'] );
+  } else {
+    $filter_by_scale = 'all';
+  }
+?>
+
+<table id="imageMaps" class="sortable">
+  <thead>
+    <th class="sorttable_nosort" style="width: 5%;"><svg width="30" height="20" viewBox="0 0 30 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+<rect x="0.5" y="0.809814" width="29" height="18" rx="9" stroke="#10355F" stroke-opacity="0.9"/>
+</svg>
+</th>
+    <th class="sorttable_nosort" style="width: 55%;"><?php esc_html_e('Description'); ?></th>
+    <th style="width: 20%;"><?php esc_html_e('Province'); ?></th>
+    <th style="width: 20%;"><?php esc_html_e('Last Updated'); ?></th>
+  </thead>
+  <tbody>
+  <?php foreach ($args['pdfs'] as $post) { ?>
+    <?php setup_postdata($post); ?>
+    <?php $related_analysis_table = beyondparallel_get_posts_using_attachment($post->ID); ?>
+    <?php 
+      $scale = get_field('scale', $post);
+      if ('all' == $filter_by_scale) { ?>                    
+        <tr>  <!-- tr -->
+        <!-- Gets all related analysis -->
+          <?php get_related_analysis($related_analysis_table); ?>
+        </tr> <!-- close tr -->
+    <?php } elseif ($scale == $filter_by_scale){ ?>
```

Comment:
this conditional branch has the same result as the `if`. do we need it? or can we just call `get_related_analysis($related_analysis_table);` ?

Reply:
I think we need it because the condition is different. The `elseif` condition is checking if the scale of the current pdf is equal to the selected scale and shows only those pdfs. FYI the var  `$scale` will never have the value `all`